apps/aecontract/src/aect_sophia.erl

```diff
@@ -12,8 +12,7 @@
 -include("../../aecontract/include/hard_forks.hrl").
 -include("../../aecontract/src/aect_sophia.hrl").
 
--export([ serialize/1
-        , serialize/2
+-export([ serialize/2
         , deserialize/1
```

Comment:
This module returns data that is not opaque so it is not immediate to follow usage, though based on the nature of the change and a quick look to DB deserialization it seems there shall be no issues in loading old DB.

Reply:
And the new contract version turned out not to be necessary, we can do with the old version. So, new update code has only minor changes.

Reply:
Better!